### PR TITLE
Add Pipenv support

### PR DESCRIPTION
**Pull request by blutack**
I've added a pipenv support file for my personal sanity so anyone running from source who does not wish to install all the packages globally can do so easily.

Hopefully this would be useful to others. All functionality seems to be working so I think I got all the packages.

### CODE REVIEW

**Comment by pavel-kirienko** [comment]
Okay. Thanks. Why the lock file?

**Comment by blutack** [comment]
Sorry for the delay. The Pipfile.lock includes locked versions of the dependencies so that pipenv will always produce the same environment (unless the versions are updated obvious).
It's usually included in the repository but could be excluded if you would prefer? [1].

1: https://docs.pipenv.org/en/latest/basics/#general-recommendations-version-control

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+[dev-packages]
+
+[packages]
+numpy = "*"
+pyqt5 = "*"
+uavcan = "*"
+qtawesome = "*"
+pyserial = "*"
+pyyaml = "*"
+qtconsole = "*"
```
Comment (pavel-kirienko) [inline Pipfile]: Any chance to have these synchronized with setup.py?

**Comment by pavel-kirienko** [comment]
> he Pipfile.lock includes locked versions of the dependencies so that pipenv will always produce the same environment (unless the versions are updated obvious).
> It's usually included in the repository but could be excluded if you would prefer? [1].

Thank you for the clarification. I never used Pipenv myself for some reason.

I wonder if we could avoid repeating the deps specs in setup.py and Pipfile.

**Comment by pavel-kirienko** [comment]
Closing due to unresponsiveness.

**Pull request closed without merging**